tests/server/server/test_views.py

```diff
@@ -2,14 +2,13 @@
 from unittest.mock import Mock
 
 import pytest
-from http_types import Response, HttpMethod, Protocol
-from http_types.utils import RequestBuilder, ResponseBuilder
+from http_types import HttpMethod, Protocol
+from http_types.utils import ResponseBuilder
 from tornado.httpclient import HTTPRequest
 
-from meeshkan.server import make_mocking_app
+from meeshkan.server.server.server import make_mocking_app
```

Comment:
This starts to look a bit funny, `meeshkan.server.server.server` :)

Reply:
The package name was the same all the time. I just removed import from __init__.py because it broke some functions. We need to refactor that naming somehow nevertheless.